shared/src/municipality.ts

```diff
@@ -1,3 +1,5 @@
+import municipalityJson from "../src/static/municipalities.json";
```

Comment:
Relative imports are allowed in shared, correct? There are several instances I've seen of it in here

Reply:
Agree that there are a couple of relative imports in the `shared/src` directory so your approach is fine.